x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_alert_group_from_sequence.test.ts

```diff
@@ -146,6 +142,64 @@ describe('buildAlert', () => {
   });
 
   describe('recursive intersection between objects', () => {
+    describe('objectPairIntersection', () => {
+      test('should work with arrays with exact matches', () => {
```

Comment:
Nit: "should" is an unnecessary (and IMO very funny) verb to use in test descriptions. If you keep the "subject" of the test in mind (e.g. "buildAlert recursive intersection between objects" 😉 ), the test description describes a behavior of the subject (e.g. "works with arrays with exact matches"). This is why you commonly see `test` aliased as `it`; that lets the test orient/read as a sentence: "It (the subject) works with arrays with exact matches".

Also, "exact matches" is a little ambiguous here, maybe this is more accurate?
```suggestion
      test('returns the intersection of fields with identically-valued arrays', () => {
```